docs/doc_1_reference.md

```diff
@@ -325,391 +325,402 @@ Class | Method | HTTP request | Description
- - [ServerlessTenantEndpointUpdate](docs/ServerlessTenantEndpointUpdate.md)
- - [Slack](docs/Slack.md)
- - [SlackNotification](docs/SlackNotification.md)
- - [Snapshot](docs/Snapshot.md)
- - [SnapshotPart](docs/SnapshotPart.md)
- - [SnapshotRetention](docs/SnapshotRetention.md)
- - [SnapshotSchedule](docs/SnapshotSchedule.md)
- - [Source](docs/Source.md)
- - [SynonymSource](docs/SynonymSource.md)
- - [SystemStatus](docs/SystemStatus.md)
- - [Tag](docs/Tag.md)
- - [TargetOrg](docs/TargetOrg.md)
- - [TargetOrgRequest](docs/TargetOrgRequest.md)
- - [Team](docs/Team.md)
- - [TeamNotification](docs/TeamNotification.md)
- - [TeamResponse](docs/TeamResponse.md)
- - [TeamRole](docs/TeamRole.md)
- - [TenantRestore](docs/TenantRestore.md)
- - [TenantSnapshot](docs/TenantSnapshot.md)
- - [ThresholdViewInteger](docs/ThresholdViewInteger.md)
- - [TimeMetricThreshold](docs/TimeMetricThreshold.md)
- - [Toggle](docs/Toggle.md)
- - [TokenFilterasciiFolding](docs/TokenFilterasciiFolding.md)
- - [TokenFilterdaitchMokotoffSoundex](docs/TokenFilterdaitchMokotoffSoundex.md)
- - [TokenFilteredgeGram](docs/TokenFilteredgeGram.md)
- - [TokenFiltericuFolding](docs/TokenFiltericuFolding.md)
- - [TokenFiltericuNormalizer](docs/TokenFiltericuNormalizer.md)
- - [TokenFilterlength](docs/TokenFilterlength.md)
- - [TokenFilterlowercase](docs/TokenFilterlowercase.md)
- - [TokenFilternGram](docs/TokenFilternGram.md)
- - [TokenFilterregex](docs/TokenFilterregex.md)
- - [TokenFilterreverse](docs/TokenFilterreverse.md)
- - [TokenFiltershingle](docs/TokenFiltershingle.md)
- - [TokenFiltersnowballStemming](docs/TokenFiltersnowballStemming.md)
- - [TokenFilterstopword](docs/TokenFilterstopword.md)
- - [TokenFiltertrim](docs/TokenFiltertrim.md)
- - [TokenizeredgeGram](docs/TokenizeredgeGram.md)
- - [Tokenizerkeyword](docs/Tokenizerkeyword.md)
- - [TokenizernGram](docs/TokenizernGram.md)
- - [TokenizerregexCaptureGroup](docs/TokenizerregexCaptureGroup.md)
- - [TokenizerregexSplit](docs/TokenizerregexSplit.md)
- - [Tokenizerstandard](docs/Tokenizerstandard.md)
- - [TokenizeruaxUrlEmail](docs/TokenizeruaxUrlEmail.md)
- - [Tokenizerwhitespace](docs/Tokenizerwhitespace.md)
- - [TriggerIngestionRequest](docs/TriggerIngestionRequest.md)
- - [UpdateCustomDBRole](docs/UpdateCustomDBRole.md)
- - [UserAccessList](docs/UserAccessList.md)
- - [UserCert](docs/UserCert.md)
- - [UserNotification](docs/UserNotification.md)
- - [UserRoleAssignment](docs/UserRoleAssignment.md)
- - [UserScope](docs/UserScope.md)
- - [UserSecurity](docs/UserSecurity.md)
- - [Validation](docs/Validation.md)
- - [VictorOps](docs/VictorOps.md)
- - [VictorOpsNotification](docs/VictorOpsNotification.md)
- - [Webhook](docs/Webhook.md)
- - [WebhookNotification](docs/WebhookNotification.md)
- - [WeeklySchedule](docs/WeeklySchedule.md)
- - [X509Certificate](docs/X509Certificate.md)
+ - [AWSAutoScaling](https://github.com/mongodb/atlas-sdk-go/blob/main/docs/docs/AWSAutoScaling.md)
```

Comment:
So idea is that website will point to those right?

Reply:
Yes. I will look into putting the whole reference into the main site but I think that's probably a stretch goal right now